### ISSUE TITLE
Training LOCON encountered problems.

### ISSUE THREAD

**Issue by TLMEMO** [KohakuBlueleaf/LyCORIS#39]
When I updated Lycoris to version 0.1.4, I found that I couldn't train LOCON. The typical symptoms were very high loss rate, and the generated images were also very strange. However, I did not encounter these issues when training LoHa.
This is the sample image for testing and the training parameters.

accelerate launch "train_network.py"  --network_module=lycoris.kohya --pretrained_model_name_or_path="anime-full-vae-fixed.ckpt" --save_model_as=safetensors --caption_extension=".txt" --seed="114514" --resolution=768,768 --train_batch_size=6 --output_name="test" --train_data_dir="./input" --output_dir="./output/" --logging_dir="./logs" --network_alpha=1 --network_dim=16 --learning_rate=3e-4  --unet_lr=3e-4 --max_train_steps=4000 --save_every_n_epochs="1" --lr_scheduler="cosine_with_restarts" --lr_scheduler_num_cycles=1 --optimizer_type="adamw" --optimizer_args "weight_decay=2e-1" --max_grad_norm=1.0 --mixed_precision="bf16" --save_precision="fp16" --enable_bucket --bucket_reso_steps=64 --bucket_no_upscale --random_crop --max_token_length=150  --shuffle_caption --gradient_checkpointing --xformers --persistent_data_loader_workers --vae="vae-ft-mse-840000-ema-pruned.ckpt" --network_args "conv_dim=8" "conv_alpha=1" "algo=lora" --clip_skip=2 --sample_prompts="./prompt/test.txt" --sample_every_n_epochs=1 --lr_warmup_step=100 --network_train_unet_only --noise_offset=0.05

The parameters I just wrote were the ones I used to train LoHa. I have made some modifications to them.

![test1](https://user-images.githubusercontent.com/120395916/228745080-181246ef-bdb5-4f8b-b658-bf858f8267b6.png)
![test2](https://user-images.githubusercontent.com/120395916/228745122-90873fd0-87f3-4d19-81a8-3b4b17feff9c.png)

**Comment by KohakuBlueleaf** [comment]
We tried to reproduce this problem and cannot get any strange output..
I'm sorry that I cannot help you on this problem QQ

can you try more different setting?

**Comment by TLMEMO** [comment]
I resolved the issue by reinstalling the dependencies of the environment. Perhaps the conflict was caused by the new version of dependencies that conflicted with my existing ones.
